intezer_sdk/analyses_history.py

```diff
@@ -0,0 +1,159 @@
+import datetime
+from typing import List, Dict, Union
+
+from intezer_sdk.api import IntezerApi
+from intezer_sdk.results import Results
+
+
+class AnalysesHistory:
+    def __init__(self, api: IntezerApi):
+        self.api = api
+
+    def file_analyses_history(self, *,
+                              start_date: int,
+                              end_date: int,
+                              aggregate_view: bool = None,
+                              sources: List[str] = None,
+                              verdicts: List[str] = None,
+                              hash_data: str = None,
+                              family_names: List[str] = None,
+                              file_name: str = None,
+                              limit: int = 100,
+                              offset: int = 0
+                              ):
+        """
+        Query for file analyses history.
+        :param start_date: Date to query from.
+        :param end_date: Date to query until.
+        :param aggregate_view: Should the result be aggregated by latest
+        hash/url/computer.
+        :param sources: Filter the analyses by its source.
+        :param verdicts: Filter by the analysis's verdict
+        :param file_name: Filter by the uploaded file's name
+        :param family_names: Filter by the analysis's malicious family name
+        :param hash_data: Filter by the file's hash, in one of the following
+        formats: SHA256, SHA1 or MD5
+        :param limit: Number of analyses returned by the query.
+        :param offset: Number of analyses to skips the before beginning to
+        return the analyses.
+        :return: all file history analyses.
+        """
+        data = self._data_analyses_history(
```

Comment:
start_date and end_date are datetime.datetime in _data_analyses_history and here you get them as int,
and then in _data_analyses_history you convert them to int again..